docs/serverdev/index.rst

```diff
@@ -0,0 +1,130 @@
+Contributing to Sentry
+======================
+
+This part of the documentation guides you towards contributing to `Sentry <https://www.github.com/getsentry/sentry>`_.
+It covers the installation and configuration of required dependencies, as well as
+standard procedures of contribution.
+
+Setting up Your Development Environment
+---------------------------------------
+
+Macintosh OSX
+'''''''''''''
+
+Third Party Services
+~~~~~~~~~~~~~~~~~~~~
+To run the basic development server, you need to install both Redis and Postgresql.
```

Comment:
PostgreSQL (not going to notate this each time below, but should be applied elsewhere)